Jenkinsfile

```diff
@@ -12,6 +12,8 @@ pipeline {
         MPYL_JIRA_TOKEN = credentials('MPYL_JIRA_TOKEN')
         AWS_ACCESS_KEY_ID = credentials('AWS_ACCESS_KEY_ID')
         AWS_SECRET_ACCESS_KEY = credentials('AWS_SECRET_ACCESS_KEY')
+        AWS_ECR_ACCESS_KEY_ID = credentials('AWS_ECR_ACCESS_KEY_ID')
```

Comment:
Where are these values actually used? Because I don't see any reference to them?

Reply:
they are used inside the mpyl_config

Reply:
Maybe add it to the `mpyl_config.example` then?

Reply:
No we shouldn't add vandebron specific stuff to the example

Reply:
I know, but the example should show how different env var's can be used / are used in the real config

Reply:
I've added some non vdb specific stuff to the config example resembling this